### ISSUE TITLE
Shipment notifications

### ISSUE THREAD

**Issue by IMiGS** [openboxes/openboxes#235]
Is it possible to implement in a near future, e-mail templates for recipients receiving shipment notifications?
For example, I would like to add to e-mail attachment with Packing List pdf file which is available in shipment details  and add custom text to e-mail body.

**Comment by jmiranda** [comment]
I've been trying to find time for the email template feature for years. Your request might give me enough motivation to get it done. We actually already have a "template" table so it shouldn't be difficult to implement the custom email body, but the email attachment might be a little more tricky.

> Is there a place where to write all wanted feature requests? :)
Just keep creating tickets. :)

**Comment by IMiGS** [comment]
I  hope it will give enough motivation. :D

**Comment by jmiranda** [comment]
Yes, the motivation is there. Now I just need to find some time.
